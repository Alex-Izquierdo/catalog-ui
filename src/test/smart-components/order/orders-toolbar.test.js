import React from 'react';
import { shallow } from 'enzyme';
import { shallowToJson } from 'enzyme-to-json';
import OrdersToolbar from '../../../SmartComponents/Order/orders-toolbar';

describe('<OrdersToolbar />', () => {
  it('should render correctly', () => {
    const wrapper = shallow(<OrdersToolbar />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});
