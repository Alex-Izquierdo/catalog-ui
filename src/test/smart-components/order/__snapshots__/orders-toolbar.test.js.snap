// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OrdersToolbar /> should render correctly 1`] = `
<TopToolbar
  paddingBottom={false}
>
  <ToolbarSection
    aria-label="order-toolbar-heading"
    className={null}
  >
    <ToolbarGroup
      className={null}
    >
      <ToolbarItem
        className={null}
      >
        <TextContent
          className=""
        >
          <Text
            className=""
            component="h1"
          >
            Orders
          </Text>
        </TextContent>
      </ToolbarItem>
    </ToolbarGroup>
  </ToolbarSection>
</TopToolbar>
`;
